modules/core/src/main/java/org/apache/ignite/internal/processors/service/ServiceContextImpl.java

```diff
@@ -66,23 +66,29 @@
     /** Cancelled flag. */
     private volatile boolean isCancelled;
 
+    /** Service statistics flag. */
+    private final boolean isStatisticsEnabled;
+
     /**
      * @param name Service name.
      * @param execId Execution ID.
      * @param cacheName Cache name.
      * @param affKey Affinity key.
      * @param exe Executor service.
+     * @param statisticsEnabled Service statistics flag.
      */
     ServiceContextImpl(String name,
         UUID execId,
         String cacheName,
         Object affKey,
-        ExecutorService exe) {
+        ExecutorService exe,
+        boolean statisticsEnabled) {
         this.name = name;
         this.execId = execId;
         this.cacheName = cacheName;
         this.affKey = affKey;
         this.exe = exe;
+        this.isStatisticsEnabled = statisticsEnabled;
```

Comment:
this is redundant here. Or just rename constructor arg

Reply:
Overy other field has 'this'. Yes, they collide in the names. Should we leave the same style?